aces/Loris
# [Raisinbread] Added CNV/CPG records for candidates with SNP data

Added CNV and CPG records for candidates that already have SNP data. This is necessary so that the `CNV` and `Methylation` tabs in the Genomic Browser module can be tested.

**Note:** I can add more data if more is needed or requested.

#### Testing instructions
With a fresh install of RB or after running the patch:

1. Navigate to `Genomic Browser`.
2. See that the `CNV` and `Methylation` tabs now display and can filter data.

#### Links to related issues

* Resolves #6881 
* Partially resolves #6302
